examples/image-component-uploadcare/next.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  images: {
+    loader: 'uploadcare',
+    path: 'https://cd813a35e9a71d2f5125.ucr.io', // optional
+  },
```

Comment:
It would be nice to be able to pass custom options directly to the image loader. What do you think?